src/libsystemd-network/sd-dhcp6-client.c

```diff
@@ -384,6 +385,27 @@ int sd_dhcp6_client_set_request_mud_url(sd_dhcp6_client *client, const char *mud
         return free_and_strdup(&client->mudurl, mudurl);
 }
 
+int sd_dhcp6_client_set_request_user_class(sd_dhcp6_client *client, char **user_class) {
+        _cleanup_strv_free_ char **s = NULL;
+        char **p;
+
+        assert_return(client, -EINVAL);
+        assert_return(client->state == DHCP6_STATE_STOPPED, -EBUSY);
+        assert_return(user_class, -EINVAL);
+
+        STRV_FOREACH(p, (char **) user_class)
+                if (strlen(*p) > 0xffff)
```

Comment:
What about here? Now you're using UINT8_MAX in one place, and this here.

Reply:
Should be UINT16_MAX. Replaced now